### PR TITLE
Adding gcloud

### PR DESCRIPTION
**Pull request by kimh**

### CODE REVIEW

**Comment by kimh** [comment]
ping!

File: circleci-provision-scripts/gcloud.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+function install_gcloud() {
+        # Gcloud installation script needs to instal .config/gcloud dir under user directory
+        (cat <<'EOF'
+sudo bash -c 'curl "https://sdk.cloud.google.com" | CLOUDSDK_CORE_DISABLE_PROMPTS=1 CLOUDSDK_INSTALL_DIR=/opt bash'
+EOF
+	) | as_user bash
```
Comment (notnoopci) [inline circleci-provision-scripts/gcloud.sh]: ~~ The combination of `sudo` inside the heredoc with `as_user` (i.e. as ubuntu) outside and multiple `bash` invocations is a bit confusing.  Are they all needed?  How is it different from

```bash
curl "https://sdk.cloud.google.com" | CLOUDSDK_CORE_DISABLE_PROMPTS=1 CLOUDSDK_INSTALL_DIR=/opt bash
```

Comment (kimh) [inline circleci-provision-scripts/gcloud.sh]: Yes, it's confusing :( Let me add a better comment.

**Comment by kimh** [comment]
I kind need this now before GA, so merging.

**Pull request merged**